compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/node/SortedSet.jsNative.kt

```diff
@@ -27,6 +27,8 @@ internal actual class SortedSet<E> actual constructor(
         var index = list.binarySearch(element, comparator)
         if (index < 0) {
             index = -index - 1
+        } else {
+            return false
         }
         list.add(index, element)
```

Comment:
Do we need duplicates? Maybe @MatkovIvan knows

Reply:
In jvm, it's implemented as TreeSet. And theret the doc says "Adds the specified element to this set if it is not already present":

>     /**
     * Adds the specified element to this set if it is not already present.
     * More formally, adds the specified element {@code e} to this set if
     * the set contains no element {@code e2} such that
     * {@code Objects.equals(e, e2)}.
     * If this set already contains the element, the call leaves the set
     * unchanged and returns {@code false}.
     *
     * @param e element to be added to this set
     * @return {@code true} if this set did not already contain the specified
     *         element
     * @throws ClassCastException if the specified object cannot be compared
     *         with the elements currently in this set
     * @throws NullPointerException if the specified element is null
     *         and this set uses natural ordering, or its comparator
     *         does not permit null elements
     */
    public boolean add(E e) {
        return m.put(e, PRESENT)==null;
    }